pettingzoo/test/parallel_test.py

```diff
@@ -96,11 +103,11 @@ def parallel_api_test(par_env, num_cycles=1000):
                     agent
                 ), "action_space should return the exact same space object (not a copy) for an agent (ensures that action space seeding works as expected). Consider decorating your action_space(self, agent) method with @functools.lru_cache(maxsize=None)"
 
-            for agent, d in done.items():
+            for agent, d in terminated.items() or truncated.items():
```

Comment:
Am I wrong in assuming that this won't work?

Reply:
You are correct, I got a tad overzealous here. 

Changing to the form of `[(x[0], x[1] or y[1]) for x, y in zip({'agent_1': True, 'agent_2': True, 'agent_3': False, 'agent_4': False}.items(), {'agent_1': True, 'agent_2': False, 'agent_3': True, 'agent_4': False}.items())]` should work.